chore: correctly update package versions

sean-hill opened the pull request:
# Linear Link

https://linear.app/atomicbuilt/issue/SDK-77/js-sdk-version-set-incorrectly-in-config

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking change which cleans up code)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change impacts security

# Checklist:

- [x] New and existing tests pass locally with my changes
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have followed the [Code Review](https://www.notion.so/atomicfi/Code-Reviews-4c05c97bdbbe4d3aa3c6c72ca7e0d57f) and [Code Review Security](https://www.notion.so/atomicfi/Security-bbfb3b07c7494f70ac0f6f47120b10ef) guidelines
- [x] I have checked my code against flaws from the [OWASP Top 10](https://owasp.org/www-project-top-ten/)
    - A01:2021-Broken Access Control
    - A02:2021-Cryptographic Failures
    - A03:2021-Injection
    - A04:2021-Insecure Design
    - A05:2021-Security Misconfiguration
    - A06:2021-Vulnerable and Outdated Components
    - A07:2021-Identification and Authentication Failures
    - A08:2021-Software and Data Integrity Failures
    - A09:2021-Security Logging and Monitoring Failures
    - A10:2021-Server-Side Request Forgery

sean-hill committed:
chore: correctly update package versions

https://linear.app/atomicbuilt/issue/SDK-77/js-sdk-version-set-incorrectly-in-config

sean-hill committed:
fix: test snapshot for platform version update

https://linear.app/atomicbuilt/issue/SDK-77/js-sdk-version-set-incorrectly-in-config

brax10ward commented on index.js:
```diff
@@ -49,7 +49,7 @@ let atomicSDK = {
         platform: {
           name: 'browser',
-          sdkVersion: '3.0.4',
+          sdkVersion: '3.0.8',
```
Can we point this to the package.json version?

sean-hill committed:
chore: add claude.md

sean-hill committed:
feat: automatically use package.json version when publishing

https://linear.app/atomicbuilt/issue/SDK-77/js-sdk-version-set-incorrectly-in-config

sean-hill committed:
chore: fix test
